sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -480,6 +525,28 @@ export interface CryptographyOptions extends coreHttp.OperationOptions {}
  */
 export interface GetRandomBytesOptions extends coreHttp.OperationOptions {}
 
+/**
+ * Options for {@link KeyClient.releaseKey}
+ */
+export interface ReleaseKeyOptions extends coreHttp.OperationOptions {
+  /** A client provided nonce for freshness. */
+  nonce?: string;
+
+  /** The {@link KeyExportEncryptionAlgorithm} to for protecting the exported key material. */
+  algorithm?: KeyExportEncryptionAlgorithm;
+}
+
+/**
+ * Result of the {@link KeyClient.releaseKey} operation.
+ */
+export interface ReleaseKeyResult {
+  /** A signed object containing the released key. */
+  value: string;
+
+  /** The {@link KeyExportEncryptionAlgorithm} used when releasing the key, if provided. */
+  algorithm?: KeyExportEncryptionAlgorithm;
```

Comment:
Should this be nullable or even defined? The service doesn't return it, and the user must provide it. It's only null if they don't provide it (since we can't infer anything), and if they provided it they know the result. I suppose we do the same for crypto, but in that case the algorithms are necessary.

What's not clear from the swagger is what the default is. Therefore, should the service actually return this and, therefore, we should define this as non-null?

/cc @herveyw-msft

Reply:
The service **does** return it, you just can't see it because it is part of the JWS object in the response.